exercises/ex2.cpp

```diff
@@ -5,47 +5,23 @@
 */
 
 #include <iostream>
+#include <string.h>
 using namespace std;
 
 int main()
 {
   int week;
 
-  cout << "Enter week number(1-7): " << endl;
-  cin >> week;
-
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
-    cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
-    cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
+  const char* a[7]= {"Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"};
 
+  cout<<"Enter a week number (1-7)"<<endl;
+  cin>>week; 
+  if(week <1 || week>7){  
+    cout<<"error "<<endl;
+    return 0; 
+  } 
+  
+  cout<<a[week-1]<<endl;;
+  
   return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```